package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-places-autocomplete",
-  "version": "7.1.2",
+  "version": "7.2.0",
```

Comment:
I will bump the version when I push this to npm registry, so no need to make this change in this PR 👍

Reply:
addressed in https://github.com/hibiken/react-places-autocomplete/pull/214/commits/f25dcc7447476c849723717fe2bb56d9c34eac03